deployment/.env.example

```diff
@@ -0,0 +1,5 @@
+MQTT_USERNAME=sample_user
```

Comment:
Please put the keys also in docker-compose.yml and reference them as variables.
Then docker compose tells you that it can't find the variable, when you forgot to create the .env file. 
Also the user sees, which are all the required/possible environment variables in docker-compose.yml. All at one place and not scattered over two files.

Then you also have only one variable INFLUX_TOKEN, that is used in two environment variables of docker-compose.yml.